src/common/components/Textarea/Textarea.theme.ts

```diff
@@ -0,0 +1,39 @@
+import { type VariantProps, tv } from "tailwind-variants";
+
+export type TextareaVariants = VariantProps<typeof textareaTheme>;
+
+export const textareaTheme = tv(
+  {
+    slots: {
+      wrapper: [],
+      textarea: [
+        "flex",
+        "items-start",
+        "p-2",
+        "w-80",
+        "max-w-[30rem]",

```

Comment:
I'm thinking maybe we can just use pixel instead as I think it's easier to visualise how big it is compared to using rem. For accessibility, rem is probably more useful for text related utilities I think!

Reply:
makes sense! updated at [e467c40](https://github.com/AfterClass-io/afterclass.io-v2/pull/82/commits/e467c403cb78517ff18a0461d507d5014430f789)